clarity_ext/domain/result_file.py

```diff
@@ -19,6 +19,9 @@ def set_udf(self, name, value, from_unit=None, to_unit=None):
             value = self.units.convert(value, from_unit, to_unit)
         self.api_resource.udf[name] = value
 
+    def get_udf(self, name):
+        return self.api_resource.udf[name]
+
```

Comment:
As far as I can see, this api resource is not in synch with the domain object (result_file). The api resource has the state as  fetched first time from db.